polysemy-RandomFu/test/RandomFuSpec.hs

```diff
@@ -14,6 +14,10 @@ import           Control.Monad.IO.Class         ( liftIO )
 import qualified Data.Random                   as R
 import qualified Data.Random.Source.PureMT     as R
 
+import           System.Random.MWC
```

Comment:
Please qualify these imports or import only the functions you use.  I made an exception for the polysemy bits themselves because that's consistent with how everything in polysemy is.